### DISCUSSION TITLE
RunTime validate with control

### DISCUSSION THREAD

**Discussion by DanieleBocchino** [react-hook-form/react-hook-form#4943] [Q&A]
Hi, I'm new to using react-hook-form. I am creating a form for users with controller but I would like to show the error message as the user enters the text and when he has entered the correct field I would like it to disappear.
I was trying to use the error but it only updates after clicking submit. can you give me some advice?

**Comment by bluebill1049** [comment]
i think you just need `mode: 'onChange'` at the useForm, please refer to the doc.
